src/commands/deployment/get.ts

```diff
@@ -296,74 +142,70 @@ export const getClusterSyncStatuses = (
 };
 
 /**
- * Initializes the pipelines assuming that the configuration has been loaded
+ * Fetches pull request data for deployments that complete merge into HLD
+ * by merging a PR
+ *
+ * @param deployment deployment for which PR has to be fetched
+ * @param initObj initialization object
  */
-export const initialize = async (): Promise<InitObject> => {
-  const config = Config();
+export const fetchPRInformation = (
+  deployment: IDeployment,
+  initObj: InitObject
+): void => {
+  if (deployment.hldRepo && deployment.pr) {
+    const repo = getRepositoryFromURL(deployment.hldRepo);
+    const strPr = deployment.pr.toString();
 
-  if (
-    !config.introspection ||
-    !config.azure_devops ||
-    !config.introspection.azure ||
-    !config.azure_devops.org ||
-    !config.azure_devops.project ||
-    !config.introspection.azure.account_name ||
-    !config.introspection.azure.table_name ||
-    !config.introspection.azure.key ||
-    !config.introspection.azure.partition_key ||
-    !config.introspection.azure.key
-  ) {
-    throw Error(
-      "You need to run `spk init` and `spk deployment onboard` to configure `spk."
-    );
+    if (repo) {
+      const promise = fetchPR(repo, strPr, initObj.accountName);
```

Comment:
initObj.accessToken

Reply:
accept. Do not know why git does not bring over your changes. Maybe the code got moved around too much